app/cmd.go

```diff
@@ -517,7 +518,7 @@ Migration: %s (#%d)
 			if err != nil {
 				return errors.Wrap(err, "begin tx")
 			}
-			defer tx.Rollback()
+			defer sqlutil.Rollback(ctx, "testCmd", tx)
```

Comment:
not sure `testCmd` would be clear what's happening
```suggestion
			defer sqlutil.Rollback(ctx, "add-user", tx)
```

With something like that ^ we'd see a message like "add-user: rollback tx: some-error"